apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts

```diff
@@ -592,8 +592,8 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
             this.currentTxn.inventory.unit.rate = giddhRoundOff(Number(val), this.ratePrecision);
             this.currentTxn.inventory.unit.highPrecisionRate = this.currentTxn.inventory.unit.rate;
             this.currentTxn.convertedRate = this.calculateConversionRate(this.currentTxn.inventory.unit.rate, this.ratePrecision);
-
-            this.currentTxn.amount = giddhRoundOff((this.currentTxn.inventory.unit.rate * this.currentTxn.inventory.quantity), this.giddhBalanceDecimalPlaces);
+            this.currentTxn.amount = (this.currentTxn.inventory.unit.rate * this.currentTxn.inventory.quantity);
```

Comment:
We can't remove the `giddhRoundOff` implementation as it displays the formatted rate as per the user-selected decimal places in the settings which is stored in variable `giddhBalanceDecimalPlaces`. 

You can try below code for your issue, if it fixes this PR issue:
`this.currentTxn.amount = giddhRoundOff((this.currentTxn.inventory.unit.rate * this.currentTxn.inventory.quantity).toFixed(this.giddhBalanceDecimalPlaces), this.giddhBalanceDecimalPlaces);`